### DISCUSSION TITLE
Labeling suggestion: Maintaining the relative skeleton structure of distal body points when changing one body point

### DISCUSSION THREAD

**Discussion by Souvik-Mandal-Harvard** [talmolab/sleap#1490] [Ideas]
For a moving animal with an anatomical skeleton, often the relative distances among the body points remain more or less similar while their relative orientation changes. For example, for a walking/running human, while the knees move in consecutive frames, the relative distance between the knee and ankle, and ankle and toe remain almost similar (the skeleton remains similar). And the change of the position of the ankle and toe is very much correlated with the change of the position of the knee. The same goes for the arms or any other animal for several other types of movements. This is more apparent when we consider the distal body points of limbs.

**It will be a great addition to SLEAP labeling if changing the position of one body point automatically moves its distal body points along with it.** Users can set this as default from the *Labels* menu, or can enable this option by pressing a key on the keyboard while dragging the body point.

Thanks!

**Comment by gitttt-1234** [comment]
closing this discussion here, started a new issue to track this enhancement: https://github.com/talmolab/sleap/issues/2710
